hare3/protocol_test.go

```diff
@@ -182,8 +186,10 @@ type setup struct {
 	proposals []types.ProposalID
 }
 
-func (s *setup) thresh(v uint16) *setup {
-	s.threshold = v
+const threshold = 10
```

Comment:
Same as above, probably `// nolint` is the way to go and fix it at some later point.